### PR TITLE
Html

### LINKED ISSUE #2: HTML bootstrap

### PR DESCRIPTION
**Pull request by ghost**
Closes #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>spaceX</title>
+</head>
+    <body>
+        <div>
```
Comment (pawelsobkowski) [inline index.html:9]: It can be the `<header>` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                        <li><a>Falcon 9</a></li>
+                        <li><a>Falcon Heavy</a></li>
+                        <li><a>Starship</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <!-- Center -->
+            <div>
+                <div>
+                    <p>find your future</p>
```
Comment (pawelsobkowski) [inline index.html]:
```suggestion
                    <h1>find your future</h1>
```

Maybe `<h1>` tag?

**Review by pawelsobkowski — CHANGES_REQUESTED** [review]
I left a few comments. Please check it out 😄

File: index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>spaceX</title>
+</head>
+    <body>
+        <div>
+            <!-- Navigation Bar -->
+            <div>
```
Comment (pawelsobkowski) [inline index.html]: It can be the `<header>` tag

File: ships.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>spaceX</title>
+</head>
+    <body>
+        <div>
+            <!-- Navigation Bar -->
+            <div>
```
Comment (pawelsobkowski) [inline ships.html]: It can be the `<header>` tag

File: ships.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                        <li><a>Falcon Heavy</a></li>
+                        <li><a>Starship</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <!-- Center -->
+            <div>
+                <!-- first stage -->
+                <section>
+                    <p>ship name</p>
```
Comment (pawelsobkowski) [inline ships.html]: Maybe `<h2>` or `<h3>`?

File: ships.html [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                    <p>status: status</p>
+                    <p>long fancy description</p>
+                    <!-- overview -->
+                    <div>
+                        <p>overview</p>
```
Comment (pawelsobkowski) [inline ships.html]:
```suggestion
                        <h3>overview</h3>
```

Maybe `<h3>` tag?

File: ships.html [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+                                <p>00.00</p>
+                            </li>
+                        </ul>
+                    </div>
+                </section>
+                <!-- second stage -->
+                <section>
+                    <!-- launches -->
+                    <div>
+                        <p>launches</p>
```
Comment (pawelsobkowski) [inline ships.html]:
```suggestion
                        <h3>launches</h3>
```

**Review by pawelsobkowski — CHANGES_REQUESTED** [review]

**Review by pawelsobkowski — APPROVED** [review]

**Review by zawolanski — APPROVED** [review]

**Pull request merged**